### DISCUSSION TITLE
PBBS

### DISCUSSION THREAD

**Discussion by pfiliberti** [mfncooper/paracon#6] [Ideas]
I hated having to stand up a BPQ node and use the built in full bbs just to have a personal bbs. I hated it so much I wrote an emulator to emulate my Heathkit Pocket Packet which is a tnc2 standard tnc with small personal bbs built in. How difficult would it be to add a small pbbs into Paracon that could accept an incoming connection on a call alias like NOCALL-1?

**Comment by dranch** [comment]
I know your pain and it's strange that Linux doesn't have any form of working simple PBBS program.  My Linpac program works in a pinch but it's not a PBBS.  Have you published your emulator so that maybe we can see what it does / doesn't do so maybe a feature list could be created?

**Reply by pfiliberti** [reply]
Yes, it's [here:](https://github.com/pfiliberti/tncemu). Works under Linux just read the how to. Tech info [here:]( https://github.com/pfiliberti/tncemu/wiki/Project-Abstract).

**Comment by mfncooper** [comment]
I'm committed to the following statement in the documentation:

> [Paracon] is focused on simplicity and ease of use, and incorporates the core functionality that most packet users need without trying to include all of the bells and whistles that few would use.

As such, a PBBS won't show up as part of Paracon itself. However, I am considering some form of add-on mechanism for a future release that could be used to facilitate such a thing. Unrelated to Paracon, I also have a partial implementation of a PBBS that might serve as a test case for such an add-on mechanism.
